### DISCUSSION TITLE
"Content you are editing has changed" warning happening a lot, and refreshing doesn't help

### DISCUSSION THREAD

**Discussion by jasononeil** [community/community#14677] [Projects and Issues]
Hello

I've been getting this warning _a lot_ in the last week:

> The content you are editing has changed. Please copy your edits and refresh the page. 

![image](https://user-images.githubusercontent.com/315158/162653478-1a333ae5-5e72-4f10-bef5-f68e847bd2e1.png)

Even if its a new issue I've created, and it is unlikely other team members have made edits. It happens on issues where I have checklists, and am editing, saving, editing again etc. Some of the checklists are URLs linking to other issues.

When I refresh the page, and even do a "hard refresh", and then attempt to edit, I see the same error, no matter how many times I try. 

What does help: Signing out and signing in again 🤕 

Has anyone else noticed an increase in this? It's very frustrating

Video:

https://user-images.githubusercontent.com/315158/162654179-73ea8a55-0515-4919-af9d-a5e60eff3802.mp4

**Comment by joelberkeley** [comment]
I get this a lot, almost more often than not, and it's _really_ trying. I'll mention that this definitely only needs one person to occur. I'm working on a project on my own and I see this happen frequently.
